backend/satellite_tools/download.py

```diff
@@ -284,29 +284,33 @@ def __init__(self, retries=3, log_obj=None, force=False):
         comp = CFG.getComponent()
         initCFG("server.satellite")
         try:
-            self.threads = int(CFG.REPOSYNC_DOWNLOAD_THREADS)
-        except ValueError:
-            raise ValueError(
-                "Number of threads expected, found: '%s'" % CFG.REPOSYNC_DOWNLOAD_THREADS
-            )
-        try:
-            self.timeout = int(CFG.REPOSYNC_TIMEOUT)
-        except ValueError:
-            raise ValueError(
-                "Timeout in seconds expected, found: '%s'" % CFG.REPOSYNC_TIMEOUT
-            )
-        try:
-            self.minrate = int(CFG.REPOSYNC_MINRATE)
-        except ValueError:
-            raise ValueError(
-                "Minimal transfer rate in bytes pre second expected, found: '%s'"
-                % CFG.REPOSYNC_MINRATE
-            )
+            try:
+                self.threads = int(CFG.REPOSYNC_DOWNLOAD_THREADS)
+            except ValueError:
+                raise ValueError(
+                    "Number of threads expected, found: '%s'" % CFG.REPOSYNC_DOWNLOAD_THREADS
+                )
+            try:
+                self.timeout = int(CFG.REPOSYNC_TIMEOUT)
+            except ValueError:
+                raise ValueError(
+                    "Timeout in seconds expected, found: '%s'" % CFG.REPOSYNC_TIMEOUT
+                )
+            try:
+                self.minrate = int(CFG.REPOSYNC_MINRATE)
+            except ValueError:
+                raise ValueError(
+                    "Minimal transfer rate in bytes pre second expected, found: '%s'"
+                    % CFG.REPOSYNC_MINRATE
+                )
+        except Exception as e:
+            raise e from None
```

Comment:
This is only here to silence "During handling of the above exception, another exception occurred", right?

I think it would be cleaner (didn't think about this when I added the other `try:...except:`s) not to handle anything here and do it like this:
```python
try:
    try:
        int("foo")
    except ValueError:
        raise ValueError("nice msg") from None
finally:
    print("initCFG call")
```
Which results in:
```python
ValueError                                Traceback (most recent call last)
<ipython-input-8-89bd5c21ec97> in <module>
      3         int("foo")
      4     except ValueError:
----> 5         raise ValueError("nice msg") from None
      6 finally:
      7     print("initCFG call")

ValueError: nice msg